pkg/manifests/types.go

```diff
@@ -283,6 +283,11 @@ type NodeExporterConfig struct {
 	// For nodes with a high CPU count, you can set the limit to a low number, which  saves resources by preventing Go routines from being scheduled to run on all CPUs.
 	// However, I/O performance degrades if the `maxProcs` value is set too low, and there are many metrics to collect.
 	MaxProcs uint32 `json:"maxProcs,omitempty"`
+	// A list of network devices, as regular expressions, to be excluded from the relevant collectors configuration such as `netdev` and `netclass`.
+	// When not set, the Cluster Monitoring Operator uses a predefined list of devices to be excluded to minimize the impact on memory usage.
```

Comment:
It is better to tell the user the default pattern is `^(veth.*|[a-f0-9]{15}|enP.*|ovn-k8s-mp[0-9]*|br-ex|br-int|br-ext|br[0-9]*|tun[0-9]*|cali[a-f0-9]*)$`. 
This comment will appear in the generated documents.

Reply:
Thanks for the review! This point was actually discussed here: https://github.com/openshift/cluster-monitoring-operator/pull/1963#discussion_r1211644529 (here also I initially documented the default list)